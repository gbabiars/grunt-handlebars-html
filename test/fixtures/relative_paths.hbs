<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>

    {{#each js}}<script src="{{this}}"></script>
    {{/each}}
</head>
<body>

</body>
</html>