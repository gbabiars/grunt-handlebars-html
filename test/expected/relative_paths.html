<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>

    <script src="test/scripts/script1.js"></script>
    <script src="test/scripts/script2.js"></script>

</head>
<body>

</body>
</html>